### PR TITLE
Adding ida reverse engineering tool

### PR DESCRIPTION
**Pull request by lucien-carneiro**
I really like your tools, but I think the IDA reverse engineering tool is missing.
I don't know if it's necessary or not but I would like to propose you my pull request.
Sincerely,
Lucien

### CODE REVIEW

**Comment by ShutdownRepo** [comment]
Oooh awesome! I'm not a reverser, hence the obvious lack of IDA ^^ I'll review and merge asap

**Comment by lucien-carneiro** [comment]
Thank's very much !!

File: sources/install.sh [hunk 1]
```diff
@@ -1192,6 +1192,13 @@ function ghidra() {
   rm /tmp/ghidra_10.1.2_PUBLIC_20220125.zip
 }
 
+function ida() {
```
Comment (ShutdownRepo) [inline sources/install.sh:1195]: Could you rename that function to `install_ida` please ?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/install.sh (hunk 1 above)
Comment (lucien-carneiro) [reply sources/install.sh]: Yes no worries, it's done ;)

**Comment by ShutdownRepo** [comment]
Perfect, can you now switch the PR to target the dev branch please? I'll merge it then 😉

**Comment by lucien-carneiro** [comment]
Done ;)

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Merged, thank you for the contribution!

**Pull request merged**